api/src/wfl/executor.clj

```diff
@@ -134,6 +136,11 @@
     (throw (UserException. "Only Dockstore methods are supported."
                            {:status 400 :methodRepoMethod methodRepoMethod}))))
 
+(defn ^:private create-user-comment
+  "Create a user comment to be added to an executor submission."
+  [note {:keys [uuid] :as _workload}]
+  (str note " Workload: " uuid " host: " (env/getenv "WFL_WFL_URL")))
```

Comment:
Nit: Could instead do the following to avoid having to flank descriptors with whitespace.

```
(str/join \space [note "Workload:" uuid "host:" (env/getenv "WFL_WFL_URL)])
```